.gitignore

```diff
@@ -27,8 +27,14 @@ proguard/
 # Log Files
 *.log
 
-# OSX files
+# OS generated files #
+######################
```

Comment:
think we should cut out this comment line, to match the rest of this file's style